libs/openFrameworks/gl/ofShader.cpp

```diff
@@ -1070,6 +1077,151 @@ void ofShader::setUniforms(const ofParameterGroup & parameters) const{
 		}
 	}
 }
+
+
+//--------------------------------------------------------------
+template<typename T>
+void ofShader::setDefineConstantTemp(const string & name, T value){
+#if (!defined(TARGET_LINUX) || defined(GCC_HAS_REGEX))
```

Comment:
Will these work on VS? If not, it could be good to have the parsing done in a platform - agnostic way- otherwise nothing not even an error message happens on programs compiled with compilers that don't support gcc_regex